node/src/chain_spec.rs

```diff
@@ -122,17 +122,16 @@ fn testnet_genesis(
         }),
         dex_pallet: Some(DexPalletConfig {
             dex_treasury: DexTreasury::new(root_key, 1, 2),
+            assets: vec![0, 1, 2],
+            initial_balance: 10u128.pow(12 + 6),
+            endowed_accounts: endowed_accounts,
```

Comment:
instead of           ` endowed_accounts: endowed_accounts,`
use
            `endowed_accounts: endowed_accounts.clone(),` to be able to pass them to `pallet_balances` as well.